criteria.go

```diff
@@ -156,6 +156,33 @@ func (f *field) In(values ...interface{}) *Criteria {
 	}
 }
 
+func (f *field) Contains(elems ...interface{}) *Criteria {
+	return &Criteria{
+		p: func(doc *Document) bool {
+			fieldValue := doc.Get(f.name)
+			if s, _ := fieldValue.([]interface{}); fieldValue == nil || s == nil {
```

Comment:
Remove reflection at all. If you define `s` before the if statement, you could use it also for iteration and avoid calls to `reflect.ValueOf`, `rv.Len()` and `rv.Index().Interface()`.
```go
s, _ := fieldValue.([]interface{})
```